docs/Modules/common-modules/event-bus.md

```diff
@@ -9,4 +9,146 @@ parent: "Common Modules"
+*  `that`: optional parameter which can be used to pass context (`this`) to the callback.
+
+### Return Value
+
+_undefined_
+
+### Code Example
+```javascript
+eventBus.on('saveSVG.start', 1500, function(event) {
+  // access event type
+  event.type; // 'saveSVG.start'
+
+  // stop propagation to other listeners
+  event.stopPropagation();
+
+  // prevent event default
+  event.preventDefault();
+
+  // Some custom logic
+  // [...]
+});
+```
+
+
+## once
+
+### Usage
+
+once(events: _String_ or _Array< string >_, [priority=1000]: _number_, callback: _Function_, [that]:_Object_): _undefined_
+
+### Parameters
+
+* `events`: name of the event or _Array_ of event names.
+* `priority`: optional parameter specifying the priority in which this listener is called, where a higher number represents a higher priority and therefore leads to earlier being called. If not provided, default of `1000` will be used.
+* `callback`: function which will be invoked with `event, ...additionalArguments`. Returning anything but `undefined` from this will stop the listener propagation. Returning `false` will prevent the execution of the default action for that event.
+*  `that`: optional parameter which can be used to pass context (`this`) to the callback.
+
+### Return Value
+
+_undefined_
+
+## off
+
+### Usage
+
+off(events: _String_ or _Array< string >_, [callback]: _Function_): _undefined_
+
+### Parameters
+
+* `events`: name of the event or _Array_ of event names.
+* `callback`: optional parameter specifying the callback listener to be removed.
+
+### Return Value
+
+_undefined_
+
+### Code Example
+```javascript
+function listener(event) {
+  // access event type
+  event.type; // 'saveSVG.start'
+
+  // stop propagation to other listeners
+  event.stopPropagation();
+
+  // prevent event default
+  event.preventDefault();
+}
+
+eventBus.on('saveSVG.start', 1500, listener);
+eventBus.off('saveSVG.start', listener);
+```
+
+## createEvent
+
+### Usage
+
+createEvent(data: _Object_): _Object_
+
+### Parameters
+
+* `data`: the event data.
+
+### Return Value
+
+An _InternalEvent_ object which can be passed to [fire](#fire).
+
+## fire
+
+### Usage
+
+fire(name: _String_, [data]: _Object_, [arguments]: _...Object_): _Object_
+
+### Parameters
+
+* `name`: the event name.
+* `data`: the optional event object. If this is not already an instance of `InternalEvent`, the `fire` method will internal wrap it as such (using [`createEvent`](createEvent)).
+* `arguments`: optional additional arguments to be passed to the callback functions.
+
+### Return Value
+
+* `false` if the default action was prevented by listeners.
+* `undefined` if no listeners were registered for that event.
+* _Object_ if the invoked listener with the highest priority returns it.
+
+### Code Example
+// TODO: how many examples / how complex examples?
```

Comment:
I'd add one rather small example and one complex one.

For the complex example, maybe we can use listener chaining, as we do it in the [`HoverFix`](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/dragging/HoverFix.js#L70) (btw. one of my all-time favorites!)

```js
var count = 0;

function countListener() {
  
  // stop the counter once 'break' was fired   
  eventBus.once('break', function() {
    eventBus.off('tick', countListener);

    console.log('count', count);
    count = 0;
  });

  // count how often event was fired after first execution
  count++;
};

eventBus.on('tick', countListener);

Reply:
Great example indeed! Thanks